scenarios/02_along_the_way.cfg

```
@@ -0,0 +1,651 @@
+        [/message]
+       	{EVENT_MOVETO_HANDLER get_trident 7 6 1 merman ( _ "Controlling lightning? I always wanted to do that!") ( _ "I am not fond of magical gadgets, I will not be taking the trident.") ({STORM_TRIDENT 7 6 trident1}) yes ()}
+       	# Let me have that trident. I want to control lightning!
+       	# I’ll just leave that trident where it is.
+    [/event]
+
+	[event]
+        name=moveto
+        first_time_only=yes
+        [filter]
+            side=1
+            [filter_location]	
+            	x,y=20,27
+            	radius=2
+            		##Terrain filter not working
+            	#terrain=!,W*,M*,!,D*,G*,H*
+            [/filter_location]
+        [/filter]
+        [lift_fog]
+        	side=1
+        	x=0-45
+        	y=0-45
+        	multiturn=yes
+        [/lift_fog]
+        [remove_shroud]
+        	side=1
+        	x,y=1
+        	radius=100
+        [/remove_shroud]
+        [message]
+            speaker=unit
+            message= _ "Whoa!"
+        [/message]
+        {SCROLL_TO 20 27}
+        [delay]
+        	time=400
+        [/delay]
+        [message]
+            speaker=SupportingCharacter1
+            message= _ "That tree is huge... I've never seen anything like it."
+        [/message]
+        [message]
+            speaker=SupportingCharacter2
+            message= _ "This is no ordinary tree. I’ve seen only one like it before. It’s a greatwood, endemic to the lands of the Elves. They are unheard of on the Shimmering Isles..."
+        [/message]
+        [message]
+            speaker=Kai
+            message= _ "Strange we haven't noticed it before. It’s so close to our home..."
+        [/message]
+        {SCROLL_TO 20 27}
+        [message]
+            speaker=narrator
+            message= _ "The tree was covered in a magical shroud to protect it from outsiders."
+        [/message]
+        [message]
+            speaker=narrator
+            image=portraits/woses/ancient-wose.png
+            message= _ "Greetings friends."
+        [/message]
+        [message]
+            speaker=narrator
+            message= _ "<i>The tree looks sickly</i>."
+        [/message]
+		[message]
+            speaker=narrator
+            image=portraits/woses/ancient-wose.png
+            message= _ "There is... corruption... deep... in my roots. I know not what causes it, but perhaps... you can help."
+        [/message]
+        [message]
+            speaker=SupportingCharacter2
+            message= _ "The Naga are approaching our city, great tree. We have little time. In what way could we assist you? Your kind is not well known to me, and since heading north from the outpost I have sensed nothing amiss."
+        [/message]
+        [message]
+            speaker=narrator
+            image=portraits/woses/ancient-wose.png
+            message= _ "My roots are deep... Yours flow with the water... I will not survive... Take this seed... plant it somewhere such that my life may go on..."
```

```suggestion
            message= _ "My roots are deep... Yours flow with the water. I can sense what you cannot... I will not survive. Take this seed... plant it somewhere such that my life may go on."
```

I know this isn't exactly what we settled on in the dialogue meeting, but I like this a bit better instead of using so many ellipses.